recipes/spiceypy/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "spiceypy" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "33db27531f7e8e31ce06970231b3265fe043b608d1404c9f43b2af14508713f3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.zip
+  url: https://github.com/AndrewAnnex/SpiceyPy/archive/v{{ version }}.zip
+  sha256: {{ sha256 }}
+
+  patches:
+    - 0001-Remove-CSPICE-installation-logic.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  noarch_python: True
+
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - cspice ==66
```

Comment:
Why is this so tightly pinned?

Reply:
@AndrewAnnex will know more, because I don't know the backwards compatibility policy of CSPICE.

Reply:
so as I understand it, the naif never removes deprecated functions from cspice. New releases come every few years (they don't follow semver), which add new functions and add new functionality to existing functions (ie the parameters don't change but the accepted input does). As I understand it, backwards compatibility is strictly required, but not forward compatibility.  SpiceyPy is a wrapper for cspice, so I implement new wrapper functions as they are introduced leaving older spiceypy versions tied to older versions of cspice.